src/scripts/quests/Quests.ts

```diff
@@ -0,0 +1,257 @@
+class Quests implements Saveable {
+    saveKey = 'quests';
+
+    defaults = {
+        xp: 0,
+        refreshes: 0,
+    };
+    
+    public xp = ko.observable(0).extend({ numeric: 0 });
+    public refreshes = ko.observable(0);
+    public lastRefresh = new Date();
+    public questList: KnockoutObservableArray<Quest> = ko.observableArray();
+    public questLines: KnockoutObservableArray<QuestLine> = ko.observableArray();
+
+    constructor() {}
+
+    getQuestLine(name) {
+        return this.questLines().find(ql => ql.name.toLowerCase() == name.toLowerCase());
+    }
+
+    get level(): KnockoutComputed<number> {
+        return ko.pureComputed((): number => {
```

Comment:
I think all of these `get thing(): KnockoutComputed<T>` are going to be creating a new computed every time they are called. Is there some reason we don't want to just 
```
public level: KnockoutComputed<number> = ko.pureComputed(...
```
which would only make the computed once?

Reply:
Yeah, that makes a lot more sense